modules/web-console/web-console-common/src/main/java/org/apache/ignite/console/utils/Utils.java

```diff
@@ -197,4 +198,26 @@ public static String extractErrorMessage(String prefix, Throwable e) {
     public static <K, U> Collector<Map.Entry<K, U>, ?, Map<K, U>> entriesToMap() {
         return Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue);
     }
+
+    /**
+     * Checks that feature supported by cluster.
+     *
+     * @param featuresAttrBytes Byte array value of supported features.
+     * @param featureId Feature to check.
+     * @return {@code True} if feature is declared to be supported by cluster.
+     */
+    public static boolean gridSupports(byte[] featuresAttrBytes, int featureId) {
```

Comment:
May be `featureSupported` will be a better name than `gridSupports`?

Reply:
Remove